tests/Unirest/RequestTest.php

```diff
@@ -83,6 +83,33 @@ public function testDefaultHeader()
         $this->assertFalse(property_exists($response->body->headers, 'hello'));
     }
 
+    public function testConnectionReuse()
```

Comment:
After you implement the increasing count of connections this test case must be updated